iot-hub/Samples/service/AutomaticDeviceManagementSample/Parameters.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+using System;
+
+namespace Microsoft.Azure.Devices.Samples
+{
+    /// <summary>
+    /// Parameters for the application supplied via command line arguments.
+    /// If the parameter is not supplied via command line args, it will look for it in environment variables.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            'c',
+            "HubConnectionString",
+            HelpText = "The IoT Hub connection string. This is available under the \"Shared access policies\" in the Azure portal." +
+            "\nDefaults to environment variable \"IOTHUB_CONNECTION_STRING\".")]
+        public string HubConnectionString { get; set; } = Environment.GetEnvironmentVariable("IOTHUB_CONNECTION_STRING");
```

Comment:
When we have environment variable overrides, please follow a different pattern here. I'll call you to talk about it.

Reply:
https://github.com/Azure-Samples/azure-iot-samples-csharp/pull/316